### DISCUSSION TITLE
Pilot stress information

### DISCUSSION THREAD

**Discussion by maxime-may** [Estia-advanced-programming/pandora-public#96] [Q&A]
The information for know when the pilot is stress is written in the update section : 18/03/2020 - 20:06 : Issue #99. We consider a stress pilot if her/his heart rate jumps more than ±10 bpm.

**Comment by EHxuban11** [comment]
Just to add more information, we need to calculate the difference of bpm between two data points (see image). If the delta is bigger than ±10 bpm we consider that the pilot is stressed and we return "y" string.

For next versions, I suggest to take a bigger time window for analysing stress. For example, it could be an change of 10bpm but on a 2 seconds period, this could detect better the heart attacks.
![image](https://user-images.githubusercontent.com/59646791/225888949-f324803b-cc7c-4dc7-a74e-63c52bcff170.png)

**Comment by Balhmung** [comment]
Must the output be ONLY a letter y/n or does the system print the full sentence : "Did the pilot had a stress attack: y/n" ?

**Comment by dhmmasson** [comment]
Just one letter "y" or "n"
